src/main/java/uk/ac/ebi/eva/pipeline/model/VariantWrapper.java

```diff
@@ -26,11 +26,7 @@
     private String strand = "+";
 
     public VariantWrapper(String chromosome, int start, int end, String reference, String alternate) {
-        this(new Variant(chromosome, start, end, reference, alternate));
-    }
-
-    public VariantWrapper(Variant variant) {
-        this.variant = variant;
+        this.variant = new Variant(chromosome, start, end, reference, alternate);
```

Comment:
I thougt that we had agreed to remove this and "convert" this VariantWrapper into a EnsemblVariant. Create a Variant class just to hold those data is totally unnecesary, and even if we would want later to create a getter of Variant I would be totally against it as it would be a instance with partial information of the Variant class.